code/game/objects/obj_defense.dm

```diff
@@ -263,8 +266,9 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 	else
 		deconstruct(FALSE)
 
-//changes max_integrity while retaining current health percentage
-//returns TRUE if the obj broke, FALSE otherwise
+/**changes max_integrity while retaining current health percentage
+  *returns TRUE if the obj broke, FALSE otherwise
+  */
```

Comment:
for future reference, you can put multiple `///` lines in a row

Reply:
also due to markdown that's a single line comment, if you want two sentences you need to seperate them with a
```
*
```